### ISSUE TITLE
PhoneNumber::getGeographicalAreaCode().

### ISSUE THREAD

**Issue by TwinMist** [brick/phonenumber#10]
many thanks, that helps
what i was after if you
$phoneNumber = PhoneNumber::parse('02079460585'); 
echo $phoneNumber->getGeographicalAreaCode(); // 20  
echo $phoneNumber->getGeographicalAreaCodeLocation();  // London

$phoneNumber = PhoneNumber::parse('01993123452'); 
echo $phoneNumber->getGeographicalAreaCode(); // 01993
echo $phoneNumber->getGeographicalAreaCodeLocation();  // Witney

$phoneNumber = PhoneNumber::parse('0901254265'); 
echo $phoneNumber->getGeographicalAreaCode(); // 09
echo $phoneNumber->getGeographicalAreaCodeLocation();  // NGN
echo $phoneNumber->getNumberType();  //  premium rate

basically trying to phase everything from
https://www.ofcom.org.uk/phones-telecoms-and-internet/advice-for-consumers/advice/telephone-area-codes-tool

many thanks

**Comment by BenMorel** [comment]
Hi, it doesn't look like the upstream library [giggsey/libphonenumber-for-php](https://github.com/giggsey/libphonenumber-for-php) supports the area code location.

You can implement it with a lookup table in PHP, though. If you're only targeting the UK, it shouldn't a problem as I don't thing area codes will change very often.

Beware that `getGeographicalAreaCode()` returns the area code without the leading zero, though.
